packages/request-client.js/test/api/payment-network/erc20/proxy-contract.test.ts

```diff
@@ -0,0 +1,90 @@
+import { AdvancedLogicTypes, RequestLogicTypes } from '@requestnetwork/types';
+import ERC20ProxyContract from '../../../../src/api/payment-network/erc20/proxy-contract';
+
+import 'chai';
+import 'mocha';
+
+const chai = require('chai');
+const spies = require('chai-spies');
+const chaiAsPromised = require('chai-as-promised');
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+chai.use(spies);
+const sandbox = chai.spy.sandbox();
+
+let erc20ProxyContract: ERC20ProxyContract;
+
+const mockAdvancedLogic: AdvancedLogicTypes.IAdvancedLogic = {
+  applyActionToExtensions(): any {
+    return;
+  },
+  extensions: {
+    erc20ProxyContract: {
+      createAddPaymentAddressAction(): any {
+        return;
+      },
+      createAddRefundAddressAction(): any {
+        return;
+      },
+      createCreationAction(): any {
+        return;
+      },
+    },
+  },
+};
+
+// Most of the tests are done as integration tests in ../index.test.ts
```

Comment:
?

Reply:
Most of the features of the payment network are tested in the file `index.test.ts` by creating directly a request with the payment network.
Good point from this comment: those tests are missing

Reply:
In fact for this payment network it's not most test